web/profiles/joinup/src/Plugin/Block/AddContentBlock.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\joinup\Plugin\Block\AddContentBlock.
+ */
+
+namespace Drupal\joinup\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Url;
+
+/**
+ * Provides an 'AddContentBlock' block.
+ *
+ * @Block(
+ *  id = "add_content_block",
+ *  admin_label = @Translation("Add content"),
+ * )
+ */
+class AddContentBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $build = [
+      'collection' => [
+        '#type' => 'link',
+        '#title' => $this->t('Add collection'),
+        '#url' => Url::fromRoute('collection.propose_form'),
+        '#attributes' => ['class' => ['button', 'button--small']],
```

Comment:
Yeah it's fine, these are two of the core classes from the "Classy" theme. We can expect these to be supported in whatever theme is going to be developed for us. The standard practice is to base new themes on the "Classy" theme. Core uses the same practice, I ripped this code from `BlockListBuilder::buildBlocksForm()`.